RELEASE_NOTES.md

```diff
@@ -8,6 +8,19 @@ This document contains release notes for the current major release and all patch
 For prior releases, see
 [PRIOR\_RELEASE\_NOTES.md](PRIOR_RELEASE_NOTES.md).
 
+## Unreleased
+
+### FIPS runtime flag exposed
+
+We are updating the Chef Server package to expose a `fips` configuration flag
+in the `chef-server.rb`. Setting `fips true` and reconfiguring will start the
+server in FIPS mode. The default value of this flag is `false` except
+on systems where FIPS is enabled at the Kernel where it defaults to `true`.
+
+The only supported systems at this time for FIPS mode are RHEL. Package for
```

Comment:
No - Chef Server is built for RHEL 5, 6 and 7. We support FIPS in all those versions.